### PR TITLE
Fixes #6698: ncf init fails in webapp postinst

### PR DESCRIPTION
**Pull request by amousset**
http://www.rudder-project.org/redmine/issues/6698

### CODE REVIEW

File: rudder-webapp/SPECS/rudder-webapp.spec [hunk 1]
```diff
@@ -381,16 +379,6 @@ echo "INFO: Launching script to check if a migration is needed"
 %{rudderdir}/bin/rudder-upgrade
 echo "INFO: End of migration script"
 
-# Create and populate technique store
-if [ ! -d /var/rudder/configuration-repository ]; then mkdir -p /var/rudder/configuration-repository; fi
-if [ ! -d /var/rudder/configuration-repository/shared-files ]; then mkdir -p /var/rudder/configuration-repository/shared-files; fi
-if [ ! -d /var/rudder/configuration-repository/techniques ]; then
-	cp -a %{rudderdir}/share/techniques /var/rudder/configuration-repository/
```
Comment (amousset) [inline rudder-webapp/SPECS/rudder-webapp.spec]: Not sure about these lines, but I don't see in which conditions they can be useful.

**Comment by VinceMacBuche** [comment]
I think it happens also in debian packages, can you look into debian/postinst file please for the  same changes ? :)

File: rudder-webapp/SPECS/rudder-webapp.spec [hunk 2]
```diff
@@ -284,12 +284,6 @@ if [ $1 -eq 1 ]
 then
 		echo -e '# This sources the configuration file needed by Rudder\n. /etc/sysconfig/rudder-apache' >> /etc/sysconfig/apache2
 		echo 'DAVLockDB /tmp/davlock.db' > /etc/%{apache}/conf.d/dav_mod.conf
-
-		mkdir -p /var/rudder/configuration-repository
-		mkdir -p /var/rudder/configuration-repository/shared-files
-		touch /var/rudder/configuration-repository/shared-files/.placeholder
```
Comment (amousset) [inline rudder-webapp/SPECS/rudder-webapp.spec]: This `.placeholder` does not seem to exist on debian, can we safely skip this?

**Comment by amousset** [comment]
@VinceMacBuche You're right, PR updated!

**Comment by amousset** [comment]
PR updated

**Comment by Kegeruneku** [comment]
PR updated

**Pull request merged**